addons/point_of_sale/static/tests/tours/pos_combo_tour.js

```diff
@@ -23,31 +24,38 @@ registry.category("web_tour.tours").add("ProductComboPriceTaxIncludedTour", {
             combo.isSelected("Combo Product 8"),
             combo.isNotSelected("Combo Product 7"),
             Dialog.confirm(),
-            ...ProductScreen.selectedOrderlineHas("Office Combo"),
-            ...ProductScreen.clickOrderline("Combo Product 3"),
-            ...ProductScreen.selectedOrderlineHas("Combo Product 3", "1.0", "13.43"),
-            ...ProductScreen.clickOrderline("Combo Product 5"),
-            ...ProductScreen.selectedOrderlineHas("Combo Product 5", "1.0", "18.67"),
-            ...ProductScreen.clickOrderline("Combo Product 8"),
-            ...ProductScreen.selectedOrderlineHas("Combo Product 8", "1.0", "30.00"),
-
+            inLeftSide([
```

Comment:
Why this change ? How it was working before ?

Reply:
Each call to `selectedOrderlineHas` was calling `inLeftSide`, which unselect the order line on mobile (cause it goes to the cart then go back to product screen), so by wrapping every steps inside one `inLeftSide` it doesn't unselect the order line each time